.circleci/workflows.yml

```diff
@@ -1789,7 +1789,7 @@ jobs:
       PLATFORM: linux
     machine:
       # using `machine` gives us a Linux VM that can run Docker
-      image: ubuntu-2004:202111-02
+      image: ubuntu-2004:2024.05.1
```

Comment:
needs to be updated since `find-up` update in the `module-api` system test is now on version 7 and cannot be installed with node 16. `ubuntu-2004:202111-02` ships with node 16, as `ubuntu-2004:2024.05.1` ships with node 20.13.0